website/docs/linkerfatalwarnings.md

```diff
@@ -0,0 +1,25 @@
+Specifies specific linker warnings that should be interpreted as errors.
+
+```lua
+linkerfatalwarnings { "warnings" }
+```
+
+### Parameters ###
+
+`warnings` is a list of warnings to interpret as errors.
+
+For Visual Studio, the MSC warning number should be used to specify the warning. On other compilers, the warning should be identified by name.
+
+In addition, Premake provides two special values to turn on all compiler and linker warnings.
+
+| Value   | Description                   |
+-------------------------------------------
+| All | Treat all linker warnings as errors   |
+
+### Applies To ###
+
+Project configurations.
+
+### Availability ###
```

Comment:
Adding the example section with filter usage would be fine IMO.

Reply:
Ok, not a bad idea, I'll come up with a meanigful one (i.e. one that has a translation into other toolsets)

Reply:
Added some simple examples